release.nix

```diff
@@ -158,6 +158,9 @@ let
           substitute ${./scripts/install-nix-from-closure.sh} $TMPDIR/install \
             --subst-var-by nix ${toplevel} \
             --subst-var-by cacert ${cacert}
+
+          shellcheck -e SC1090 $TMPDIR/install
```

Comment:
SC1090: it doesn't know what `. $var/file` means or resolves to, so it can't follow to `/file` and figure out if your code is valid.
SC1091: similar to above, it can't source the file and statically check it.
SC2002: "Useless cat. Consider 'cmd < file | ..' or 'cmd file | ..' instead." except if I did `sudo ... < thefile` it complains that sudo doesn't affect pipes.